pkg/games/playground/src/Playground.ts

```diff
@@ -11,38 +11,25 @@ export class Playground extends Game<PlaygroundProps> {
       ...props,
       systems: [
         new RenderSystem({ renderer: props.renderer }),
-        new DebugSystem({ renderer: props.renderer })
+        new DebugSystem({ renderer: props.renderer }),
+        new InputSystem({ renderer: props.renderer })
       ]
     });
     this.init();
   }
 
   init = async () => {
-    // ball
-    this.addEntity(new Entity({
-      id: "ball", renderer: this.props.renderer, networked: false,
-      components: {
-        renderable: new Renderable({
-          renderer: this.props.renderer,
-          debuggable: true,
-          pos: { x: 50, y: 50 },
-          zIndex: 1,
-          container: new Text("🏀", { fill: "#FFFFFF", fontSize: 16 })
-        })
-      }
-    }));
-
     // fpsText
     this.addEntity(new Entity({
       id: "fpsText", renderer: this.props.renderer, networked: false,
       components: {
         renderable: new TextBox({
           renderer: this.props.renderer,
-          cameraPos: { x: 5, y: 5 },
+          cameraPos: { x: -35, y: 5 },
```

Comment:
moved FPS text to right side of the screen

fullscreen and debug buttons now on the left